Gemfile

```diff
@@ -1,8 +1,10 @@
-source 'https://rubygems.org'
 
-gem 'github-pages', group: :jekyll_plugins
+source 'https://rubygems.org'
+gem 'github-pages', '>=104', group: :jekyll_plugins
 
 group :development, :test do
   gem 'rake'
   gem 'anemone'
+  gem "html-proofer"
+  gem 'mdl', '~> 0.4.0'
```

Comment:
Unless you specifically need this version, you can drop the version and the Gemfile.lock will contain a specific version.